### PR TITLE
COM-00: Fix Postgres no user name specified error

### PR DESCRIPTION
**Pull request by gabrielncorreia**

### CODE REVIEW

File: mod/dbs.js [hunk 1]
```diff
@@ -9,27 +9,29 @@ module.exports = () => {
...
     const pool = new Pool({
-      user: pgSecret["username"],
-      password: pgSecret["password"],
-      connectionString: connectionString,
-      statement_timeout: parseInt(process.env.STATEMENT_TIMEOUT) || 10000,
+      database: 'map',
+      user: pgSecret['username'],
+      password: pgSecret['password'],
+      host,
+      port: 5432,
```
Comment (mbarcia) [inline mod/dbs.js:33]: This is less flexible as it only allows to choose the host, and fixes the port, db, and any other param that we might want to change through the envvar. But it's just an observation.

File: mod/dbs.js [hunk 2]
```diff
@@ -9,27 +9,29 @@ module.exports = () => {
...
         return aurora(key.split('_')[1], process.env[key]);
     });
 
-  async function postgres(key, connectionString) {
-    let secretClient = new SecretsManager({region: process.env.AWS_DEFAULT_REGION});
-    let secretId = `${process.env.XYZ_ENV}/xyz/pg`;
-    const pgSecret = await secretClient.getSecretValue({SecretId: secretId}).promise().then((data) => {
+  async function postgres(key, host) {
+    const secretClient = new SecretsManager({region: process.env.AWS_DEFAULT_REGION});
+    const secretId = `${process.env.XYZ_ENV}/xyz4/pg`;
```
Comment (mbarcia) [inline mod/dbs.js]: Last time I checked, xyz secret was there. Any reason for changing it?

**Review by mbarcia — APPROVED** [review]

File: mod/dbs.js (hunk 1 above)
Comment (gabrielncorreia) [reply mod/dbs.js:33]: I'm fixing that right now. We are going to use `host` when in the Pod and `connectionString` when local.

File: mod/dbs.js (hunk 2 above)
Comment (gabrielncorreia) [reply mod/dbs.js]: This was a typo. Already fixing it.

File: mod/dbs.js (hunk 1 above)
Comment (mbarcia) [reply mod/dbs.js:33]: I'm afraid that would be an anti-pattern. Just use one envvar for all envs and it should work.

**Review by mbarcia — CHANGES_REQUESTED** [review]
Please remove that piece of code where you ask for the ENV, it's an anti-pattern.

**Review by mbarcia — APPROVED** [review]

**Pull request merged**